Updated Freoduo keymap

FilipParyz opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description
Update to the handwired split keyboard.

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

drashna commented on keyboards/handwired/freoduo/keymaps/default/keymap.c:
```diff
@@ -144,43 +140,43 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
   switch (keycode) {
-    case LOWER:
+    case _LOWER:
```
You don't want to use the layer numbers here.  These start at 0, and increment up.  Several of these actually have pre-defined values that you're overlapping (KC_NO, and KC_TRNS)

drashna commented on keyboards/handwired/freoduo/keymaps/default/keymap.c:
```diff
@@ -144,43 +140,43 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
       if (record->event.pressed) {
         layer_on(_LOWER);
-        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+        update_tri_layer(_LOWER, _RAISE, _FN);
```
You may not be able to access the _FN layer this way.  Layer processing is done from the highest numbered layer, down. _FN is set to be lower than the raise or lower layers, so it will be processed after those layers.

FilipParyz commented:
I've simplified the keymap and removed the tri-layer completely. I don't need it 🤷‍♂️

Okay, I see there are some lint checks not passing as I don't have license headers included. I'll fix that.

FilipParyz commented:
@drashna Hi, can I ask for a review? :)
